webproject/src/test/java/com/google/sps/data/MessageTest.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.data;
+
+import static org.junit.Assert.assertEquals;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test class for Message.
+ */
+public class MessageTest {
+    private static final String USER = "test user";
+    private static final String MESSAGE = "test message";
+    private static final String TYPE = "test type";
+
+    private Message message;
+
+    @Before
+    public void setUp() {
+        message = new Message(USER, MESSAGE, TYPE);
+    }
+
+    @Test
+    public void getUser_returnsUser() {
+        assertEquals(USER, message.getUser());
+    }
+
+    @Test
+    public void getTime_returnsTime() {
+        assertEquals(new SimpleDateFormat("HH:mm").format(new Date()), message.getTime());
+    }
```

Comment:
The code here is time sensitive so it might not pass 100% of the time.

Reply:
when https://github.com/arjavibahety/SpsTeam8/pull/56/files#diff-cc8f223e1684e083dde9ec51abb485e1
 got merged, you should implement a FakeTimestampUtil class to be injected
the FakeTimestampUtil class should be set-able to return a specific date time

Reply:
Updated